unblob/file_utils.py

```diff
@@ -65,6 +66,35 @@ def find_first(
         bytes_searched += chunk_size - compensation
 
 
+def iterate_file(
+    file: io.BufferedIOBase,
+    start_offset: int,
+    size: int,
+    # default buffer size in shutil for unix based systems
+    buffer_size: int = 64 * 1024,
```

Comment:
Yeah, it would be a cool idea.

It has a  value we would use:
```python
COPY_BUFSIZE = 1024 * 1024 if _WINDOWS else 64 * 1024
```

Sadly, I just realized `shutil` does not offers it, just use it internally. (so I can not use `shutil.COPY_BUFSIZE`).

Should it remain as it is? Should I add this line and write a comment that it is from `shutil`?....

Reply:
We discussed it offline and I created a separate commit for it - it has some `pyright` problems, but we can use that import.